kolibri/auth/management/commands/fullfacilitysync.py

```diff
@@ -89,9 +95,30 @@ def create_superuser_and_provision_device(self, username, dataset_id):
             device_settings.save()
 
     def handle_async(self, *args, **options):
+        # validate url that is passed in
+        try:
+            URLValidator()((options['base_url']))
+        except ValidationError:
+            print('Base-url is not valid. Please retry command and enter a valid url.')
+            return
```

Comment:
I think it's better to use a `sys.exit(1)` call, so you are sure that execution halts and that the calling process knows that the management command has failed? It depends of course a bit on how you intend the management command to be used, but in general they should be called from command line and not with `call_command`.